doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -561,6 +561,18 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
+* :ref:`rreq_readme` library:
+
+  * Added
+
+    * Added the Ranging Service Ranging Requestor library for Channel Sounding.
+
+* :ref:`rrsp_readme` library:
+
+  * Added
+
+    * Added the Ranging Service Ranging Responder library for Channel Sounding.
+
```

Comment:
```suggestion
* Added the :ref:`rreq_readme` and :ref:`rrsp_readme` libraries.
```
Like this, but remove the changelog entry from this PR and add it as a suggestion to my https://github.com/nrfconnect/sdk-nrf/pull/19240.